main.go

```diff
@@ -684,6 +687,10 @@ func main() {
 		Name: "container_gpu_allocation",
 		Help: "container_gpu_allocation GPU used",
 	}, []string{"namespace", "pod", "container", "instance", "node"})
+	PVAllocation := prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "pod_pvc_allocation",
```

Comment:
Should we be prefixing all our new metrics with `_kubecost`? That's what we're doing for recording rules now.

Reply:
I did this for consistency with the others-- we should rename them all at once if we're going to do that.

Reply:
Let's leave as is then for now.